### PR TITLE
Added DI into DomainAliasConrtoller.

### PR DESCRIPTION
**Pull request by BasilSnowman**
Added DI into DomainAliasConrtoller.

### CODE REVIEW

File: domain_alias/src/Controller/DomainAliasController.php [hunk 1]
```diff
@@ -24,7 +24,7 @@ public function addAlias(DomainInterface $domain) {
     // the parent domain entity.
     $values['domain_id'] = $domain->id();
     // @TODO: ensure that this value is present in all cases.
-    $alias = \Drupal::entityTypeManager()->getStorage('domain_alias')->create($values);
+    $alias = $this->entityManager->getStorage('domain_alias')->create($values);
```
Comment (andypost) [inline domain_alias/src/Controller/DomainAliasController.php]: lib/Drupal/Core/Controller/ControllerBase.php:54

File: domain_alias/src/Controller/DomainAliasController.php [hunk 2]
```diff
@@ -39,7 +39,7 @@ public function addAlias(DomainInterface $domain) {
    *   A render array as expected by drupal_render().
    */
   public function listing(DomainInterface $domain) {
-    $list = \Drupal::entityTypeManager()->getListBuilder('domain_alias');
+    $list = $this->entityManager->getListBuilder('domain_alias');
```
Comment (andypost) [inline domain_alias/src/Controller/DomainAliasController.php]: do not use deprecated

File: domain_alias/src/Controller/DomainAliasController.php (hunk 1 above)
Comment (BasilSnowman) [reply domain_alias/src/Controller/DomainAliasController.php]: domain/domain/src/Controller/DomainControllerBase.php:52

Comment (andypost) [reply domain_alias/src/Controller/DomainAliasController.php]: deprecated!!!!

File: domain_alias/src/Controller/DomainAliasController.php [hunk 3]
```diff
@@ -24,7 +24,7 @@ public function addAlias(DomainInterface $domain) {
     // the parent domain entity.
     $values['domain_id'] = $domain->id();
     // @TODO: ensure that this value is present in all cases.
-    $alias = \Drupal::entityTypeManager()->getStorage('domain_alias')->create($values);
+    $alias = $this->entityTypeManager->getStorage('domain_alias')->create($values);
```
Comment (andypost) [inline domain_alias/src/Controller/DomainAliasController.php:27]: Use method instead of variable

Comment (andypost) [reply domain_alias/src/Controller/DomainAliasController.php:27]: Protected property may not be initialized

**Review by andypost — COMMENTED** [review]
Filed https://github.com/agentrickard/domain/pull/339

**Pull request closed without merging**